plugins/chat/assets/stylesheets/common/base-common.scss

```diff
@@ -617,6 +617,14 @@ html.has-full-page-chat {
       grid-template-rows: calc(
         var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px)
       );
+
+      .footer-nav-ipad & {
+        grid-template-rows: calc(
+          var(--chat-vh, 1vh) * 100 - var(--header-offset, 0px) -
+            var(--footer-nav-height, 0px)
+        );
+      }
```

Comment:
We were doing hacks in desktop/chat-composer because of this mistake I think, if the size is correct we don't need to add custom margin when there's `footer-nav-ipad` element